packages/ag-charts-community/src/chart/series/series.ts

```diff
@@ -249,13 +251,19 @@ export class HighlightStyle {
     readonly text = new TextHighlightStyle();
 }
 
-export class SeriesTooltip {
+export class SeriesTooltip<P extends AgSeriesTooltipRendererParams> {
     @Validate(BOOLEAN)
     enabled: boolean = true;
 
     @Validate(OPT_BOOLEAN)
     showArrow?: boolean = undefined;
 
+    @Validate(OPT_FUNCTION)
+    renderer?: (params: P) => string | AgTooltipRendererResult = undefined;
+
+    @Validate(OPT_STRING)
+    format?: string = undefined;
```

Comment:
So I'm just wondering if in the cases where we don't support `format` we could ensure that trying to set it will still a console warn?

The hacky way to do this would be to NOT assign `undefined `here so that the field isn't present at runtime by default, and then in the constructor passing an option to control an initial assignment to `undefined` if we're expecting users to be able to set it?

Reply:
Discussed offline: We'll add `formatter` support universally.